torch/fx/experimental/proxy_tensor.py

```diff
@@ -59,8 +60,12 @@ def wrap_with_proxy(e, proxy):
     else:
         return real_out
 
+from torch.testing._internal.composite_compliance import check_metadata_consistency
```

Comment:
imports at the top

Reply:
Also I don't think we should be using testing code here.

Reply:
I think we should refactor this for generic use on wrapper tensor subclasses. I'm gonna comment this out and leave it as a todo for a follow-up PR though.